bin/build-db

```diff
@@ -11,11 +11,20 @@ console.log(`Building database for the ${process.env.NODE_ENV} env…`);
 
 let foodProcesses, textileProcesses;
 if (process.env.NODE_ENV === "test") {
-  foodProcesses = "public/data/food/processes_impacts.json";
-  textileProcesses = "public/data/textile/processes_impacts.json";
+  foodProcesses = process.env.FOOD_PROCESSES_IMPACTS_PATH;
+  textileProcesses = process.env.TEXTILE_PROCESSES_IMPACTS_PATH;
+
+  if (!foodProcesses || !textileProcesses) {
+    console.error(
+      "\n🚨 ERROR: For the tests to work properly, you need to specify FOOD_PROCESSES_IMPACTS_PATH and TEXTILE_PROCESSES_IMPACTS_PATH env variables. They need to point to the detailed versions of the processes files. Please, edit your .env file accordingly.",
+    );
+    console.error("-> Exiting the build-db process.\n");
+    process.exit(1);
+  }
 } else {
-  foodProcesses = "public/data/food/processes.json";
-  textileProcesses = "public/data/textile/processes.json";
+  foodProcesses = process.env.FOOD_PROCESSES_IMPACTS_PATH || "public/data/food/processes.json";
+  textileProcesses =
+    process.env.TEXTILE_PROCESSES_IMPACTS_PATH || "public/data/textile/processes.json";
 }
```

Comment:
My understanding is that if we're not in a test environment, eg. we're in production, which defines `FOOD_PROCESSES_IMPACTS_PATH`, we'll build the db using the detailed impacts version instead of the non-detailed version — which is not what we want I think. Am I misunderstanting something?